tests/RestfulViewEntityMultiLingualTestCase.test

```diff
@@ -147,8 +147,8 @@ class RestfulViewEntityMultiLingualTestCase extends RestfulCurlBaseTestCase {
     $response = $handler->get($id);
     $result = $response[0];
 
-    $this->assertEqual(trim(strip_tags($result['text_single'])), $text1, 'Entity view has correct result for "text_single" in language "' . $langcode . '".');
-    $this->assertEqual(trim(strip_tags($result['text_multiple'][0])), $text1, 'Entity view has correct result for the first value of "text_multiple" in language "' . $langcode . '".');
-    $this->assertEqual(trim(strip_tags($result['text_multiple'][1])), $text2, 'Entity view has correct result for the second value of "text_multiple" in language "' . $langcode . '".');
```

Comment:
@e0ipso
I understand, do you have any suggestions how to make this one to work?

Since the format has changed for translated fields, these tests will always fail because they don't relate to the `$langcode`.